packages/modules/web-viewer-react/src/services/Initializer.ts

```diff
@@ -133,6 +133,8 @@ export class WebInitializer {
           WebInitializer._initializing = false;
           WebInitializer._cancel = undefined;
         });
+    } else if (IModelApp.initialized && !this._initialized) {
```

Comment:
the cases here should be 

`else if (this._initializing) {
   return this._initialized;
}
else {
// throw the error here.....the error message should notify the user to use the useWebViewInitializer hook in their app instead of calling IModelApp.startup
}
}`